practice-one/pages/index.html

```diff
@@ -64,7 +64,7 @@ <h2 class="section-title">What We Do</h2>
           </div>
           <div class="card-body">
             <h2 class="card-title">UX Design</h2>
-            <h6 class="card-details">Mobile App, Website</h6>
+            <h3 class="card-details">Mobile App, Website</h3>
             <p class="card-text">
```

Comment:
Class **card-text** => with this way of naming, I will imagine this is a normal text. But based on design, it's describing some info for the service UX Design. So will be better if you change it to **service-description**, how about your idea?

Reply:
I want to create a frame card that I can use for all the cards on this page. So I use a common name